modules/anki/hm.nix

```
@@ -0,0 +1,339 @@
+{
+  config,
+  lib,
+  mkTarget,
+  pkgs,
+  ...
+}:
+mkTarget {
+  name = "anki";
+  humanName = "Anki";
+
+  configElements =
+    { colors }:
+    {
+      programs.anki.addons = [
+        (pkgs.ankiAddons.recolor.withConfig {
```

Consider using `lib.singleton`:

```suggestion
      programs.anki.addons = lib.singleton (
        pkgs.ankiAddons.recolor.withConfig {
```